source/ed/data.cpp

```diff
@@ -113,7 +113,19 @@ void Data::build_block_id() {
                 // This is not supposed to happen
                 // @TODO: Add a parameter to avoid too long connection
                 // they can be for instance due to bad data
```

Comment:
Maybe a bit of explanation could help later debug (ascii-art?)

Reply:
I've done a little further done :+1: